DAQ/src/StrawHitRecoFromFragments_module.cc

```diff
@@ -314,7 +315,12 @@ void art::StrawHitRecoFromFragments::analyze_tracker_(
           ADCWFIter maxiter;
           pmp = _shrUtils.peakMinusPedWF(trkDataPair.second,srep,maxiter);
         }
-        _shrUtils.createComboHit(-1,chCol, shCol, caloClusters, pbtOffset, sid, tdc, tot, pmp, trackerStatus,  srep, tt);
+
+        // temporary hack, FIXME
+        mu2e::EventWindowMarker ewm;
+        ewm._spillType = mu2e::EventWindowMarker::onspill;
+        ewm._eventLength = 1695.0;
+        _shrUtils.createComboHit(ewm,-1,chCol, shCol, caloClusters, pbtOffset, sid, tdc, tot, pmp, trackerStatus,  srep, tt);
```

Comment:
shall we create an interface that uses a `struct` instead of having a long list of inputs? I can do it in a separate MR if people agree on doing that.